src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -164,7 +164,7 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
                 on-click="entityMoreInfo"
                 alt="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
                 title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
-                path="[[informationOutlineIcon()]]"
+                .path="[[informationOutlineIcon()]]"
```

Comment:
this is polymer, not lit.

Reply:
🤦‍♂️ sorry about that. Hopefully one day everything will be moved to lit.